### DISCUSSION TITLE
Automate the task creation

### DISCUSSION THREAD

**Discussion by divyagothi13** [madrecha/jamku#935] [General]
### Feature Request

As of now Jamku does not have the automatic task creation facility; Although they have started a scheduler while will "remind" user about the task creation. But this feature is almost similar to "Reminder" where we can set any type of reminder.

Task creation process is time-consuming. Because if one task takes approx 30 seconds and suppose we have 100 monthly tasks & for 12 months it would be 1200 task  which would come around to 10 hours which is huge.

This time can be saved by just creating one feature. 
This can be acheived by changing the way the task is created. When a user creates a task for the first time , Jamku should ask the frequency of task and repeatation feature like we see in the google reminder.

This is sort of example, repeatation feature can be set to default for all so that only Monthly/Daily/Quarterl/Halfyearly/Yearly frequency is left to be added by the user.

I have been in an organisation where we have used there inhouse task management system, where we used to have this feature.
Feel free to contact me on  9638211219

### Use-Case

This will save my time each and every year.

**Comment by adarshmadrecha** [comment]
I understand your request for "Automated task creation". However, this feature brings more "Bad things" then Good.
In fact, I know 2 practice management software who had this feature and later removed it due to various issues.

Let me share the most important reasons for not bringing this feature.
1. Human angle - If someone manually creates the task, he is most likely to ensure it's completion. Hence, we suggest managers or seniors should create the tasks instead of Admin in the office.
2. Reliability angle - In a rare scenario, if the automated task creation did not happen due to 
   a. Misconfiguration
   b. Bug in software itself
Then the **reliability** of the system itself is gone for a toss. Having manual task creation ensures confidence in the system.
3. Once the offices get used to Jamku, it does not take too much effort to create the tasks. It might take more effort to ensure correct configuration for automatic task creation than creating the tasks itself.

**Comment by divyagothi13** [comment]
Hi,

Thanks for the reply!
But  I disagree with your view. You can have a survey run for this and you
will come to know what is the truth behind it.


Thanks & Regards

**Comment by adarshmadrecha** [comment]
> I disagree with your view. You can have a survey run for this and you will come to know what is the truth behind it.

That's a nice suggestion. Here's the link for the Survey.
https://forms.gle/P3Wp6gxDYy59Fo7q7


![image](https://user-images.githubusercontent.com/11911938/152644842-414bd44d-6089-4bf6-8a58-a5f775a942b6.png)

**Comment by adarshmadrecha** [comment]
Here's the survey result 

![image](https://user-images.githubusercontent.com/11911938/152644930-1a8b6fad-33c2-4656-bdb6-a357b6c55d21.png)
![image](https://user-images.githubusercontent.com/11911938/152644980-d4aa57f3-e495-43a0-871b-2f7b962d7892.png)
![image](https://user-images.githubusercontent.com/11911938/152644997-d956fadb-413b-430c-bf63-7c55ba352889.png)
